dotCMS/src/integration-test/java/com/dotcms/graphql/business/GraphqlAPITest.java

```diff
@@ -631,40 +631,33 @@ public void testGetSchema_BothSidedRelationshipFields(final RELATIONSHIP_CARDINA
 
     }
 
-    @DataProvider
-    public static List<Object> dataProviderEEContentTypes() throws Exception {
-        // data provider needs stuff to get initialized because of API access
-        IntegrationTestInitService.getInstance().init();
+    @Test
+    public void testGetSchema_GivenNoEELicense_EnterpriseTypesShouldNotBeAvailableInSchema() throws Exception{
 
         // filter only Enterprise content types
-        final List<ContentType> eeTypes = APILocator
+        List<ContentType> eeTypes = APILocator
                 .getContentTypeAPI(APILocator.systemUser()).findAll().stream()
                 .filter((type)->type instanceof EnterpriseType).collect(Collectors.toList());
 
-        // returns a List of Tuple (typeName, baseType)
-        return eeTypes.stream().map((type)->
-                new Tuple2<>("my"+type.variable(), type.baseType())
-        ).collect(Collectors.toList());
-    }
-
-    @Test
-    @UseDataProvider("dataProviderEEContentTypes")
-    public void testGetSchema_GivenNoEELicense_EnterpriseTypesShouldNotBeAvailableInSchema(
-            final Tuple2<String, BaseContentType> testCase) throws Exception{
-        ContentType customType = null;
-
-        try {
-            // create custom persona type. 1=typeName, 2=BaseType
-            customType = createType(testCase._1,
-                    testCase._2);
-
-            runNoLicense(() -> {
-                final GraphQLSchema schema = APILocator.getGraphqlAPI().getSchema();
-                assertNull(schema.getType(testCase._1));
-            });
-        } finally {
-            if(customType!=null) {
-                APILocator.getContentTypeAPI(APILocator.systemUser()).delete(customType);
+        List<Tuple2<String, BaseContentType>> eeTypesList = eeTypes.stream().map((type)->
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'eeTypesList' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4826945)